common-lib/common/signal/test_marshall_json.libsonnet

```diff
@@ -0,0 +1,135 @@
+local signal = import './signal.libsonnet';
+local test = import 'jsonnetunit/test.libsonnet';
+
+local jsonSignals =
+  {
+    aggLevel: 'group',
+    groupLabels: ['job'],
+    instanceLabels: ['instance'],
+    filteringSelector: 'job="integrations/agent"',
```

Comment:
nit; We've moved all of our external facing assets (where possible) to use `alloy` in place of `agent`. Would be nice to update that here.